### PR TITLE
Replace fmt.Print with log.Error

### PR DESCRIPTION
**Pull request by vparfonov**
Signed-off-by: Vitalii Parfonov <vparfono@redhat.com>

### Description
Replace fmt.Printf() with log.Error() or log.Info() 

/cc @jcantrill 
/assign @jcantrill 

/cherry-pick

### CODE REVIEW

File: internal/cmd/forwarder-generator/main.go [hunk 1]
```diff
@@ -65,5 +64,5 @@ func main() {
 		log.Error(err, "Unable to generate log configuration")
 		os.Exit(1)
 	}
-	fmt.Println(generatedConfig)
+	log.Info("Generated Configuration", "generatedConfig", generatedConfig)
```
[the remark this answers is not in the document]
Comment (vparfonov) [reply internal/cmd/forwarder-generator/main.go:67]: ok

**Review by periklis — COMMENTED** [review]
/lgtm

**Review by jcantrill — COMMENTED** [review]
/approve

**Pull request merged**